packages/react-adminer/src/components/FavoriteList.tsx

```diff
@@ -35,29 +34,49 @@ const FavoriteList: React.FC<Props> = inputEntityName => {
 					setLocalStorageData(finalObjContent);
 				} else {
 					setIsEmpty(true);
+					setLocalStorageData(undefined);
 				}
 				return;
 			} catch {
 				//
 			}
 		}
 		setIsEmpty(true);
-	}, []);
+	}, [reducerValue]);
+
+	const deleteFavoriteFilter = (record: any): void => {
+		const favourites = localStorageData.favourites.filter((i: any) => i.name !== record.name);
```

Comment:
Co když budu mít více stejných názvů ? To nebude fungovat, chtělo by to tam přidat nějaký ID už při ukládání a podle něj to dělat.

Reply:
Přidání oblíbenýho filteru neumožňuje přidat 2 filtry se stejným 'name'.